### PR TITLE
Remove-DbaDatabaseSafely - New improvement and some minor fixes

### PR DESCRIPTION
**Pull request by ClaudioESSilva**
This will support the need that someone ask to @SQLDBAWithABeard some time ago in a conference.
If the database last full backup took more than X minutes (I choose 10) a question will popup asking if user want to continue or skip that database.

FIxes:
- Change `$DestinationCredential` from `[Object]` to `[System.Management.Automation.CredentialAttribute()]`
- Uses `Get-DbaBackupHistory -LastFull` to know how much time the last full backup took. If more than 10 minutes asks if user want to continue or skip that database
- Number of retries to refresh is 4 but the intent was 5. Changed to match that
- Moved the returned [PSCustomObject] to inside the foreach loop.

Question:
- It is OK as it is or should we include a new parameter like `-StopIfLastBackupTookMoreThan` (we can get a bigger parameter name :-P) to check based on that value?

### CODE REVIEW

File: functions/Remove-DbaDatabaseSafely.ps1 [hunk 1]
```diff
@@ -149,7 +149,8 @@ If there is a DBCC Error it will continue to perform rest of the actions and wil
 		[object[]]$Database,
 		[parameter(Mandatory = $false)]
 		[object]$Destination = $sqlinstance,
```
Comment (wsmelton) [inline functions/Remove-DbaDatabaseSafely.ps1]: This should be set to `[DbaInstanceParameter]` as well...while you are updating.

File: functions/Remove-DbaDatabaseSafely.ps1 [hunk 2]
```diff
@@ -642,7 +671,7 @@ If there is a DBCC Error it will continue to perform rest of the actions and wil
 			
 			$refreshRetries = 1
 			
-			while (($destserver.databases[$dbname] -eq $null) -and $refreshRetries -lt 5) {
+			while (($destserver.databases[$dbname] -eq $null) -and $refreshRetries -lt 6) {
 				Write-Verbose "Database $dbname not found! Refreshing collection"
```
Comment (wsmelton) [inline functions/Remove-DbaDatabaseSafely.ps1]: This should be updated to `Write-Message`.

**Comment by potatoqualitee** [comment]
hey @ClaudioESSilva  - you got a moment to update this so we can get it in the Wednesday beta?

**Comment by ClaudioESSilva** [comment]
Thanks for the review @wsmelton 
If you can, please review the changes and approve it.

**Comment by wsmelton** [comment]
Test fail on a parameter is missing in the help.

**Comment by potatoqualitee** [comment]
thanks @ClaudioESSilva ~

**Pull request merged**